docs/stencil-docs/developing-further/stored-credit-card-management.mdx

```diff
@@ -84,6 +84,110 @@ curl -O https://raw.githubusercontent.com/leeBigCommerce/cornerstone/54f5681a6a1
 ## Update `en.json`
 At minimum, a translation for the `en` needs to be added to `lang/en.json: [Pull Request #1603](https://github.com/bigcommerce/cornerstone/pull/1603/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-b0d4c1fc9d8d2a5a213b27a72cf6c9fe). Add translations for other locales as needed.
 
+## Manage stored account payments
+
+After adding a [stored payment method](https://support.bigcommerce.com/s/article/Enabling-Stored-Payment-Methods?language=en_US), shoppers can use the Storefront account payments microapp to manage their stored payment instruments. The Storefront account payments provide a payment method form with billing address fields. Currently, the microapp works only with Adyen.
+
+### Install the microapp
+
+To install the Storefront account microapp to your theme, inject the `account_payments` variable into your template using `add-payment-method.html`.
+
+```handlebars showLineNumbers copy
+{{#if account_payments}}
+    {{{ account_payments }}}
+    <script>
+        window.BigCommerce = window.BigCommerce || {};
+    </script>
+{{/if}}
+```
+`account_payments` will provide the `<div>` element where you can render Storefront account payments and attach the renderAccountPayments() method to the window.BigCommerce object that displays the Storefront account payments app if the selected payment provider is already implemented.

```

Comment:
```suggestion
The `account_payments` variable includes the `<div>` element that renders the Storefront account payments microapp. You can use a script to attach the `renderAccountPayments()` method to the `window.BigCommerce` object. If the selected payment provider is already implemented, calling `renderAccountPayments()` renders the Storefront account payments microapp.

```

@vitalii-koshovyi I am not sure where the developer attaches `renderAccountPayments()`. Do they add the method on line 99 so that they can call it from a listener?

Reply:
@slsriehl 
The `storefront-account-payments` microapp automatically attaches the `renderAccountPayments()` method to the `window.BigCommerce` object.

The `account_payments` variable is just a `<div>` with the script tag containing the microapp url.
```
<div id="bc-account-payments"></div>

<script type="text/javascript" src="{{ filePath to storefront-account-payments build  }}"></script>
```

So after the script is loaded, storefront-account-payments render method `renderAccountPayments()` is attached to the  `window.BigCommerce`. So theme developer can use it in their script to initialize  the app.

Reply:
@bc-traciporter fyi